akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -171,6 +173,63 @@ final class ClusterClientSettings(
       heartbeatInterval, acceptableHeartbeatPause, bufferSize, reconnectTimeout)
 }
 
+/**
+ * Declares a super type for all events emitted by the `ClusterClient`
+ * in relation to contact points being added or removed.
+ * Events are published to the Akka event stream (system.eventStream).
+ * Given that the event bus is local only, there is no serialization
+ * of them that is required.
+ */
+sealed trait ContactPointChange extends NoSerializationVerificationNeeded {
+  val contactPoint: ActorSelection
```

Comment:
the rationale is that `ActorSelection` is something you send messages to, but the purpose of this API is to get information of the contact points so that it can for example be stored